password/impl/src/main/java/org/wildfly/security/password/impl/ThreadLocalSecureRandom.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * JBoss, Home of Professional Open Source
+ *
+ * Copyright 2024 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.security.password.impl;
+
+import java.security.SecureRandom;
+import java.util.function.Supplier;
+
+public class ThreadLocalSecureRandom implements Supplier<SecureRandom> {
```

Comment:
This looks great thank you, one further minor change - could you please drop the public modifier as this is only accessed from classes in the same package.

Later we may want to move to a more central location but for now this is great for these classes.

Reply:
Sure, I made it package local.